format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSinkConfig.java

```diff
@@ -59,15 +65,47 @@
 
   public void validate() {
     IdUtils.validateId(referenceName);
-    if (suffix != null && !containsMacro("suffix")) {
+    if (suffix != null && !containsMacro(NAME_SUFFIX)) {
       new SimpleDateFormat(suffix);
     }
-    if (!containsMacro("format")) {
+    if (!containsMacro(NAME_FORMAT)) {
```

Comment:
this shouldn't have been here before, format is not macro enabled so this will always be true

Reply:
![image](https://user-images.githubusercontent.com/14131070/65291543-b1293580-db08-11e9-973e-92cc661b5269.png)

Reply:
that's weird... it's not annotated as a macro above.

Reply:
I will remove macros for format property.